### PR TITLE
[SPARK-16651][PYSPARK][DOC] Make `withColumnRenamed/drop` description more consistent with Scala API

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
## What changes were proposed in this pull request?

`withColumnRenamed` and `drop` is a no-op if the given column name does not exists. Python documentation also describe that, but this PR adds more explicit line consistently with Scala to reduce the ambiguity.
## How was this patch tested?

It's about docs.

### CODE REVIEW

File: python/pyspark/sql/dataframe.py [hunk 1]
```diff
@@ -1388,6 +1388,7 @@ def withColumn(self, colName, col):
     @since(1.3)
     def withColumnRenamed(self, existing, new):
         """Returns a new :class:`DataFrame` by renaming an existing column.
+        This is a no-op if schema doesn't contain existingName.
```
Comment (rxin) [inline python/pyspark/sql/dataframe.py]: existingName is not an argument.

Comment (dongjoon-hyun) [inline python/pyspark/sql/dataframe.py]: Oops.

**Comment by dongjoon-hyun** [comment]
Thank you for quick review, @rxin . I fixed it.

**Comment by srowen** [comment]
While we're here, I note that `drop` methods have the same caveat in the Scala docs. I guess we can apply them to the Python docs as well.

**Comment by dongjoon-hyun** [comment]
Oh, I see. No problem.

**Comment by dongjoon-hyun** [comment]
@srowen . I added the same note to `drop` and amended both description a little.

**Comment by srowen** [comment]
Merged to master/2.0

**Comment by dongjoon-hyun** [comment]
Thank you for review and merging, @srowen and @rxin .

**Pull request closed without merging**